### PR TITLE
Added URL component

### PR DESCRIPTION
**Pull request by AndyKarseras**
# minor

Implemented to resolve BPSO-22018 URL input component for Bunsen/Frost

### CODE REVIEW

**Comment by sglanzer-fire** [comment]
I don't think this belongs in `core` - maybe a `frost-url-test` addon?

File: docs/frost-urlinput.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# frost-text
```
Comment (sandersky) [inline docs/frost-urlinput.md]: It looks like this file has lots of references to `frost-text` that should actually be `frost-url-input`.

File: addon/templates/components/frost-text.hbs [hunk 2]
```diff
@@ -10,5 +10,5 @@
 }}
 {{#if showClear}}
   {{!frost-icon class='clear' icon='close'}}
-  <a class="clear" {{action "clear"}}>x</a>
+  <a class="clear" {{action "clear"}}>1</a>
```
Comment (sandersky) [inline addon/templates/components/frost-text.hbs]: This should be reverted.

File: addon/styles/_frost-url-input.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.urlIcon {
+  display: block;
+  vertical-align: middle;
+  > .frost-icon {
+    ertical-align: middle;
```
Comment (sandersky) [inline addon/styles/_frost-url-input.scss]: Typo

File: addon/components/frost-url-input.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      // Verify URL - potentially change to a solid regex in the future
+      if ((!host) || (!host.toLowerCase().startsWith('http://')) && (!host.toLowerCase().startsWith('https://'))) {
+        this.set('urlFormatError', true)
+        return
+      }
+
+      this.send('clear')
+      this.set('isLoading', true)
+
+      let that = this
```
Comment (sandersky) [inline addon/components/frost-url-input.js]: If you use arrow functions below you shouldn't need a variable named `that`.

**Comment by sandersky** [comment]
The component name is `frost-url-input` but in other places in this PR you have files and references to `frost-urlinput`. I'd try to be consistent and keep the name the same throughout.

**Comment by AndyKarseras** [comment]
To be merged to ember-frost-fields

**Pull request closed without merging**